include/linker/common-rom.ld

```diff
@@ -109,12 +101,11 @@
 #endif
 
 #if defined(CONFIG_NET_L2_PPP)
-	SECTION_PROLOGUE(net_ppp_proto,,)
+	SECTION_PROLOGUE(_ppp_protocol_handler,,)
```

Comment:
Can you please describe briefly why this needs to change to _protocol_handler?

Reply:
The system did not crash in short testing (more needed with different hw to fully verify). Also the said macro will have Z_DECL_ALIGN() call which I hope will set the proper alignment automatically.